aws/session/credentials_test.go

```diff
@@ -63,11 +63,31 @@ func setupCredentialsEndpoints(t *testing.T) (endpoints.Resolver, func()) {
 
 	stsServer := httptest.NewServer(http.HandlerFunc(
 		func(w http.ResponseWriter, r *http.Request) {
-			w.Write([]byte(fmt.Sprintf(
-				assumeRoleRespMsg,
-				time.Now().
-					Add(15*time.Minute).
+			if err := r.ParseForm(); err != nil {
+				w.WriteHeader(500)
+				return
+			}
+
+			form := r.Form
+
+			switch form.Get("Action") {
+			case "AssumeRole":
+				w.Write([]byte(fmt.Sprintf(
+					assumeRoleRespMsg,
+					time.Now().
+						Add(15*time.Minute).
+						Format("2006-01-02T15:04:05Z"))))
```

Comment:
Does this have access to the format constant used by the timestamp deserializer for these fields?